ietf/templates/doc/bofreq/bofreq_template.md

```diff
@@ -30,9 +30,9 @@ To allow evaluation of your proposal, please include the following items:
 
 ## Links to the mailing list, draft charter if any, relevant Internet-Drafts, etc.
    - Mailing List: https://www.ietf.org/mailman/listinfo/example
-   - Draft charter: https://datatracker.ietf.org/doc/charter-ietf-EXAMPLE/
+   - Draft charter: {{ settings.IDTRACKER_BASE_URL }}/doc/charter-ietf-EXAMPLE/
```

Comment:
These _could_ be changed to use urlreverse, but I don't know if the benefit is as high as elsewhere.